inspirehep/modules/fixtures/users.py

```diff
@@ -28,48 +28,96 @@
             description='users with editing capabilities'
         )
+        ds.create_role(
+            name='hermescurator',
+            description='curator for HERMES Internal Notes'
+        )
+        ds.create_role(
+            name='hermescoll',
+            description='HERMES Collaboration access to Internal Notes'
+        )
+    db.session.commit()
+
+
+def init_users():
+    """Sample users, not to be used in production."""
+    ds = current_app.extensions['invenio-accounts'].datastore
+    superuser = Role.query.filter_by(name='superuser').one()
+    cataloger = Role.query.filter_by(name='cataloger').one()
+    hermes_curator = Role.query.filter_by(name='hermescurator').one()
+    hermes_collections = Role.query.filter_by(name='hermescoll').one()
+    with db.session.begin_nested():
         ds.create_user(
             email='admin@inspirehep.net',
             password=hash_password("123456"),
             active=True,
-            roles=[superuser_role]
+            roles=[superuser]
         )
         ds.create_user(
             email='cataloger@inspirehep.net',
             password=hash_password("123456"),
             active=True,
-            roles=[cataloger_role]
+            roles=[cataloger]
         )
-        db.session.add(ActionRoles(
-            action='superuser-access',
-            role=superuser_role
-        ))
-        db.session.add(ActionRoles(
-            action='admin-access',
-            role=superuser_role)
-        )
-        db.session.add(ActionRoles(
-            action='workflows-ui-admin-access',
-            role=cataloger_role)
+        ds.create_user(
+            email='hermescataloger@inspirehep.net',
+            password=hash_password("123456"),
+            active=True,
+            roles=[hermes_curator, hermes_collections]
         )
-        db.session.add(ActionRoles(
-            action='admin-holdingpen-authors',
-            role=cataloger_role)
+        ds.create_user(
+            email='johndoe@inspirehep.net',
+            password=hash_password("123456"),
+            active=True
         )
+    db.session.commit()
+
 
+def init_permissions():
+    superuser = Role.query.filter_by(name='superuser').one()
+    cataloger = Role.query.filter_by(name='cataloger').one()
+    hermes_collections = Role.query.filter_by(name='hermescoll').one()
+    db.session.add(ActionRoles(
+        action='superuser-access',
+        role=superuser)
+    )
+    db.session.add(ActionRoles(
+        action='admin-access',
+        role=superuser)
+    )
+    db.session.add(ActionRoles(
+        action='workflows-ui-admin-access',
+        role=cataloger)
+    )
+    db.session.add(ActionRoles(
+        action='admin-holdingpen-authors',
+        role=cataloger)
+    )
+    db.session.add(ActionRoles(
+        action='view-restricted-collection',
+        argument='HERMES Internal Notes',
+        role=hermes_collections)
+    )
     db.session.commit()
+
+
+def init_all_users():
```

Comment:
Since this function is still initializing the permissions, the previous name was better. But what about adding an `init_all_fixtures` function that calls all the fixtures (collections, files, users, roles, permissions...) in the right order?